ex03.ex

```diff
@@ -59,8 +59,15 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def odd_even([]),             do: []
+  def odd_even([head | tail]),  do: [odd_even_atom(head) | odd_even(tail)]
```

Comment:
Good naming.

I'm not sure you need the `is_integer` test, as `Integer.is_even` would presumably fail if not. In that case, you could simplify to

~~~ elixir
 def odd_even_atom(number) when Integer.is_even(number) do
    :even
  end
  def odd_even_atom(number) when is_integer(number) do
    :odd
  end
~~~